src/types/config.d.ts

```diff
@@ -1,18 +1,2 @@
-interface NetworkIdBasedConfig<T> {
-  [networkId: number]: T
-}
-
-export type TcrConfig = NetworkIdBasedConfig<{
-  listId?: number
-  contractAddress: string
-}>
-
-export type DexPriceEstimatorConfig = NetworkIdBasedConfig<string>
-
-export type TheGraphApiConfig = NetworkIdBasedConfig<string>
-
-export interface Config {
-  tcr?: TcrConfig
-  dexPriceEstimator: DexPriceEstimatorConfig
-  theGraphApi: TheGraphApiConfig
-}
+import { Config } from 'types/config'
+declare let CONFIG: Config
```

Comment:
Because you are doing `import` it changes scope of every `declare`, do
```suggestion
import { Config } from 'types/config'

declare global {
  const CONFIG: Config
}

```

Reply:
Let me try that 😍

Reply:
It works! However, for some reason, the Global config and the other globals don't get along together, I fixed it using your tips, this is why I keep it in separate files 🤷‍♂️

Reply:
So the genral approachis this
```ts
//something.d.ts
declare let GLOBAL_1: type
declare const GLOBAL_2 = 123456
```
variables get declared in global namespace
BUT
```ts
import something something
// or
export something
// make all declarations module scoped
// so must do
declare global {
  declare let GLOBAL_1: type
  declare const GLOBAL_2 = 123456
}
```
So in a sentence: if need `import/export` then `declare global`